### DISCUSSION TITLE
Using <segment>'s within 'Image Edit Batcher' Tool

### DISCUSSION THREAD

**Discussion by imap86** [mcmonkeyprojects/SwarmUI#496] [Q&A] [answered]
Hey there,

i really love the Image Edit Batcher Tool and a was trying to add a e.g. <segment:face,> prompt
to the prompt line, but it seems i cant set it up the right way or this is not supported.
It always processes the whole image.

Anyone knows how i can use segments on a batch of already generated images?

e.g. Usecase: 
a big Folder of Images generated with Model A -> reprocess only faces of all images with Model B with Image Edit Batcher 

Greetings form Austria

**Comment by mcmonkey4eva** [comment, marked as answer]
It should mostly work the same as doing that regularly. Note, in particular, you have to use Init Image with Creativity at 0.
